app/controllers/medical_records_controller.rb

```diff
@@ -22,6 +28,8 @@ def edit
   # POST /medical_records or /medical_records.json
   def create
     @medical_record = MedicalRecord.new(medical_record_params)
+    @medical_record.image.attach(params[:medical_record][:image]) if params[:medical_record][:image]
```

Comment:
I think line 30 is enogth to attach the image, as you can see in this example https://guides.rubyonrails.org/active_storage_overview.html#has-many-attached.